components/admin/orgs/initialValues.ts

```diff
@@ -1,16 +1,15 @@
-
 const defaultValues: OrganisationForm = {
-    id: -1,
-    name: '',
-    description: '',
-    inviteLink: '',
-    // photoUpload: null,
-    isAdminOrg: false,
-    isInvisible: false,
-    isInactive: false,
-    category: '',
-    igHead: -1,
-    otherMembers: [],
-  }
+  id: -1,
+  name: '',
+  description: '',
+  inviteLink: '',
+  // photoUpload: null,
+  isAdminOrg: false,
+  isInvisible: false,
+  isInactive: false,
+  category: 'Sports',
```

Comment:
Parth - To test

Reply:
Fixed and tested in 867b13d723b48bca01ba2b189aaaa51285bfb597